proxies-es6/demo.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+// Proxying a normal object
+var target = {};
+var handler = {
+  get(receiver, name = 'world') {
+    return `Hello, ${name}!`;
+  }
+};
+
+var p = new Proxy(target, handler);
+ChromeSamples.log(p.folks);
+
+// Proxying a function object
+function sum(a, b) {
+  return a + b;
+}
+
+var handler2 = {
+  apply: function(target, thisArg, argumentsList) {
+    console.log(`Calculate sum: ${argumentsList}`);
+    return target.apply(thisArg, argumentsList);
+  }
+};
+
+var proxy = new Proxy(sum, handler2);
```

Comment:
You've got `handler` and `handler2` as variable names. It could be clearer to use `proxy` and `proxy2` for the corresponding proxies.